internal/capability/operand_cleanup.go

```diff
@@ -0,0 +1,67 @@
+package capability
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/opdev/opcap/internal/operator"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// OperandCleanup removes the operand from the OCP cluster in the ca.namespace
+func (ca *capAudit) OperandCleanUp() error {
+	logger.Debugw("cleaningUp operand for operator", "package", ca.subscription.Package, "channel", ca.subscription.Channel, "installmode", ca.subscription.InstallModeType)
+
+	if len(ca.customResources) > 0 {
+		obj := &unstructured.Unstructured{Object: ca.customResources[0]}
+
+		// using dynamic client to create Unstructured objests in k8s
+		client, err := operator.NewDynamicClient()
+		if err != nil {
+			log.Println(err)
+		}
+
+		var crdList apiextensionsv1.CustomResourceDefinitionList
+		err = ca.client.ListCRDs(context.TODO(), &crdList)
+		if err != nil {
+			logger.Error(err.Error())
```

Comment:
Also another point of return right? If we can't list the CRDs wouldn't it break the code since we need to create the GVRs?